### PR TITLE
KEYCLOAK-6514 Common approach to writing performance tests

### PR DESCRIPTION
**Pull request by mstruk**

### CODE REVIEW

File: testsuite/performance/tests/src/test/scala/keycloak/DefaultSimulation.scala [hunk 1]
```diff
@@ -1,30 +1,18 @@
...
+import keycloak.DefaultScenarioBuilder._
+
 import org.keycloak.performance.TestConfig
 
+
 /**
   * @author Radim Vansa &lt;rvansa@redhat.com&gt;
   * @author Marko Strukelj &lt;mstrukel@redhat.com&gt;
   */
 class DefaultSimulation extends Simulation {
```
Comment (tkyjovsk) [inline testsuite/performance/tests/src/test/scala/keycloak/DefaultSimulation.scala]: Should we also rename the the `DefaultSimulation` class (and `DefaultScenarioBuilder`) to something more descriptive, like: `OIDCLoginLogoutSimulation`, `BasicOIDCSimulation`, or maybe just `OIDCSimulation`?

**Review by tkyjovsk — COMMENTED** [review]
The changes look good. The test cases seem much more readable now. I tried out the changes locally and both scenarios seem to work fine.

File: testsuite/performance/tests/src/test/scala/keycloak/DefaultSimulation.scala (hunk 1 above)
Comment (mstruk) [reply testsuite/performance/tests/src/test/scala/keycloak/DefaultSimulation.scala]: BasicOIDCSimulation sounds good to me. I'll change it.

Comment (tkyjovsk) [reply testsuite/performance/tests/src/test/scala/keycloak/DefaultSimulation.scala]: Can you please also update class names in the READMEs?

**Comment by mstruk** [comment]
I changed the DefaultSimulation name to BasicOIDCSimulation, and also changed references in READMEs

**Review by tkyjovsk — APPROVED** [review]

**Review by pdrozd — APPROVED** [review]

**Pull request merged**